site/en/docs/extensions/mv3/architecture-overview/index.md

```diff
@@ -48,252 +54,353 @@ as the most important files and the capabilities the extension might use.
 }
 ```
 
+### Toolbar icon {: #icons }
+
 Extensions must have an icon that sits in the browser toolbar. Toolbar icons allow easy access and
 keep users aware of which extensions are installed. Most users will interact with an extension that
-uses a [popup][2] by clicking on the icon.
+uses a [popup][docs-popup] by clicking on the icon, like in the [Getting Started
+example][sample-getting-started].
+
+{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/ku5Z8MMssgw6MKctpJVI.png", alt="Getting started
+Popup", width="187", height="153" %}
+<!-- TODO: Show examples of the MV3 getting started tutorial extensions -->
+
+### Background service worker {: #background_script }
```

Comment:
@dotproto Do you have any thoughts on this? 

A. Background service worker
B. Background script
C. Service worker

I was reading the [Migrate from Background Pages to Service Workers](https://developer.chrome.com/docs/extensions/mv3/migrating_to_service_workers/), and looks like we mostly refer to it as service worker 🤔.

Reply:
If this is a standard service worker as defined for a web page, then 'service worker' is the only thing you can use. If it differs in some way, I would suggestion staying away from the term 'service worker'.

Reply:
@dotproto Did you say you wanted to go from background service worker to extension service worker?

In the migration doc, we call them [background service worker](https://developer.chrome.com/docs/extensions/mv3/intro/mv3-migration/#background-service-workers). Should we also change it here to **extension service worker**?

Reply:
Sorry for the delay. I used the phrasing "background service worker" early on in the Manifest V3 doc process due to the extension platform's historical use of "background scripts" and "background page". In this context, the background prefix was a signal that we were referring to the part of an extension's architecture that was typically responsible for coordinating work rather than displaying a UI surface.

As time passed, I realized that this phrasing was inaccurate because, as @jpmedley said, "background service worker" is redundant. In order to differentiate between open web and extension usages, I currently prefer to use "extension service worker" or simply "service worker"

> @dotproto Do you have any thoughts on this?
> 
> A. Background service worker
> B. Background script
> C. Service worker

I'd go with Option C. I'd avoid Option A for the previously outlined reasons. I'd avoid Option B because that term has historically been synonymous with background pages. (Developers could declare their "background scripts" by setting `background.scripts` in manifest.json. Under the hood Chrome would automatically create a page that they were injected into.)

Reply:
> If this is a standard service worker as defined for a web page, then 'service worker' is the only thing you can use. If it differs in some way, I would suggestion staying away from the term 'service worker'.

It does differ in that extension developers do not manually control the installation and update process. Additionally, the way service workers are used inside extensions is different. On the web, service workers are primarily used as a network proxy and caching layer for web pages. In extensions, service workers handle events, background logic, and coordinate other parts of the extension's experience.

We may be able to deemphasize the term "service worker", but I don't think we can completely avoid it because that is the foundational technology on which the Manifest V3 background script execution is built and it's enshrined in the manifest key extension developers use to register that background context (`background.service_worker`).